cli/find.php

```diff
@@ -30,6 +30,17 @@
 require_once($CFG->libdir.'/clilib.php');
 require_once($CFG->libdir.'/adminlib.php');
 
+$url_types = [
+    'course' => function($record) {return (new moodle_url('/course/view.php', array('id' => $record->id)))->out();},
+    'page' => function($record) {return (new moodle_url('/course/view.php', array('id' => $record->id)))->out();},
```

Comment:
this should not link to the course it should link to the page inside the course

eg /mod/page/view.php?id=123